CITATION.cff

```diff
@@ -158,11 +158,11 @@ authors:
     "orcid": "https://orcid.org/0000-0003-0590-7843"
 
 cff-version: "1.0.3"
-date-released: 2020-10-12
-doi: "10.5281/zenodo.3387139"
+date-released: 2021-2-8
+doi: "10.5281/zenodo.4081401"
```

Comment:
This DOI should not be changed, it's a generic DOI for all versions.

Reply:
If you go there it says that there is a newer version available. That's why I proposo to update it

Reply:
It doesn't for me, could it be your browser cache or something? This is the DOI listed on zenodo as the generic DOI for all versions:
> Cite all versions? You can cite all versions by using the DOI [10.5281/zenodo.3387139](https://doi.org/10.5281/zenodo.3387139). This DOI represents all versions, and will always resolve to the latest one. [Read more](http://help.zenodo.org/#versioning).

Reply:
A Firefox + Google issue: firefox does not resolve the DOI url and searchs google, which in turn presents as the first result not the actual generic DOI but a specific version one

Reply:
I am reverting the change